src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.scss

```diff
@@ -57,8 +57,8 @@ $stacking-order: (
 }
 
 .newDesignLanguage {
-  .CenterContainer + .RightContainer {
```

Comment:
I have a feeling we'll still need this

Reply:
The approach I took was applying the margin-right to the CenterContainer, so that we can treat all of the other buttons as a group (no matter the size of the browser, there will always be a spacing to the button groups and the centerContainer). The sizing issue with the More Filters button occurred because when the size is reduced, the moreFilters is apart of its own container (the margin-left was originally only applied to the right container).

Reply:
Right but if there are no filters buttons then the search bar wouldn't span the full width of its container. I think it's safer to use the `+` selector and apply a left margin when they exist

Reply:
We could also do `only-child` to revert the `margin-right`.